src/NHibernate/Mapping/ByCode/IModelInspector.cs

```diff
@@ -7,7 +7,7 @@ public interface IModelInspector
 	{
 		bool IsRootEntity(System.Type type);
 		bool IsComponent(System.Type type);
-		bool IsEntity(System.Type type);
+		bool IsEntity(System.Type type, bool explicitlyDeclared = false);
```

Comment:
Given that default parameters aren't currently used in the code base this makes sense to expose two different methods, will update